features/26-mfa/api.md

```diff
@@ -188,7 +188,15 @@ Activate the just created inactive TOTP authenticator.
 
 ## Query parameters
 
-- `otpauth_uri`: The otpauth URI
+- `otpauth_uri`: The [key URI](https://github.com/google/google-authenticator/wiki/Key-Uri-Format).
+- `account_name`: The [account name](https://github.com/google/google-authenticator/wiki/Key-Uri-Format#label).
+- `secret`: The [Base32 encoded secret](https://github.com/google/google-authenticator/wiki/Key-Uri-Format#secret).
+- `issuer`: The [issuer](https://github.com/google/google-authenticator/wiki/Key-Uri-Format#label).
+
+If `otpauth_uri` is given, it is used as is. Otherwise `otpauth_uri` is generated from `account_name`, `secret` and `issuer`.
+
+This endpoint accepts `secret` instead of `authenticator_id` because this endpoint is supposed to be used directly,
```

Comment:
Oops, I totally didn't get it before you spell it out.

Ummm, do we really want to encourage that? Coz I think embed the secret in image URL is not a good practices and we shouldn't encourage users to do so... (since URL could be tracked by browsers / extensions / server side logs etc)

I think it is better to display the QR code by:

1. Users call the APIs, get some sort of `data:image/png` format, and render it, OR
2. Get the png from a stateful image URL

Reply:
Or you have researched and it is commonly done in this way? @louischan-oursky